test/ng/directive/inputSpec.js

```diff
@@ -4211,6 +4211,22 @@ describe('input', function() {
       expect(inputElm[0].getAttribute('value')).toBe('something');
     });
 
+    it('should update the dom "value" to "" and attribute to null when the binding is set to undefined', function() {
+      var inputElm = helper.compileInput('<input type="text" ng-value="value">');
+
+      $rootScope.$apply('value = \'something\'');
+
+      expect(inputElm[0].value).toBe('something');
+      expect(inputElm[0].getAttribute('value')).toBe('something');
+
+      $rootScope.$apply(function() {
+        delete $rootScope.value;
+      });
+
+      expect(inputElm[0].value).toBe('');
+      expect(inputElm[0].getAttribute('value')).toBe(null);
```

Comment:
I prefer to keep the expectation on other browsers (even if it is not directly related to this PR, it is still relevant for the added test).